code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -123,11 +123,18 @@
 	list_reagents = list("epinephrine" = 30, "weak_omnizine" = 30, "salglu_solution" = 30)
 
 /obj/item/reagent_containers/hypospray/combat/nanites
-	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanites for rapid healing."
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with a cocktail of experimental combat drugs and <b>extremely</b> expensive medical nanomachines. Capable of healing almost any injury in just a few seconds."
 	icon_state = "nanites_hypo"
 	volume = 100
 	list_reagents = list("nanites" = 100)
 
+/obj/item/reagent_containers/hypospray/combat/syndicate_nanites
+	name = "medical nanite injector"
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanomachines for rapid field stabilization."
+	volume = 100
+	possible_transfer_amounts = list(1,2,3,4,5,10,15,20,25,30) // i want to let them to inject more/less depending on the situation, while keeping the flags the base combat has
```

Comment:
Are you ever going to want to select anything under 5 units in a realistic situation? I think that starting at 5 and incrementing in the same amount would be more intuitive.

Reply:
Probably not? but there's no harm in letting them finely tune it to 10.

Reply:
The base hypo already has those transfer amounts, ya? The funny one you're adding should just inherit those.

Reply:
If it's being inherited from the base type I guess it's fine. I was just thinking that a longer list of options that you won't ever use would be extra clutter.

Reply:
> If it's being inherited from the base type I guess it's fine. I was just thinking that a longer list of options that you won't ever use would be extra clutter.

Perhaps, perhaps not. What actually matters is that that the list already exists and doesn't need to be specified twice.